docs/src/main/sphinx/connector/hive.rst

```diff
@@ -537,63 +537,66 @@ In order to use a Glue catalog, ensure to configure the metastore with
 ``hive.metastore=glue`` and provide further details with the following
 properties:
 
-==================================================== ============================================================
-Property Name                                        Description
-==================================================== ============================================================
-``hive.metastore.glue.region``                       AWS region of the Glue Catalog. This is required when not
-                                                     running in EC2, or when the catalog is in a different region.
-                                                     Example: ``us-east-1``
+===================================================== ============================================================
```

Comment:
I guess this change in unnecessary.

Reply:
It's required for .rst file to parse tables properly. Number of == really matters. Without that table gets hidden.

Reply:
Every table in the table in this file follows same convention I guess. The problem I faced with this table is, my property name little too long. If I reduce it by 2 characters then I don't have to do any changes to rest.

Reply:
Changing this property name to `hive.metastore.glue.aws-credentials-providerconf` with one character less. Should solve this. Going with that :)